File: AppLovin-Android-Demo-App/src/main/java/com/applovin/apps/demoapp/rewarded/RewardedVideosDemoMenuActivity.java
package com.applovin.apps.demoapp.rewarded;

import android.content.Intent;

import com.applovin.apps.demoapp.DemoMenuActivity;
import com.applovin.apps.demoapp.DemoMenuItem;
import com.applovin.apps.demoapp.rewarded.RewardedVideosActivity;
import com.applovin.apps.demoapp.rewarded.RewardedVideosZoneActivity;

public class RewardedVideosDemoMenuActivity
        extends DemoMenuActivity
{
    @Override
    protected DemoMenuItem[] getListViewContents()
    {
        DemoMenuItem[] result = {
                new DemoMenuItem( "Basic Integration", "Quick rewarded video integration", new Intent( this, RewardedVideosActivity.class ) ),
                new DemoMenuItem( "Zone Integration", "Create different user experiences of the same ad type", new Intent( this, RewardedVideosZoneActivity.class ) )
        };
        return result;
    }
}

File: Android-Kotlin-Demo-App/src/main/java/com/applovin/apps/kotlindemoapp/rewarded/RewardedVideosDemoMenuActivity.kt
package com.applovin.apps.kotlindemoapp.rewarded

import android.content.Intent
import com.applovin.apps.kotlindemoapp.DemoMenuActivity
import com.applovin.apps.kotlindemoapp.DemoMenuItem

class RewardedVideosDemoMenuActivity : DemoMenuActivity()
{
    override fun getListViewContents(): Array<DemoMenuItem> = arrayOf(
            DemoMenuItem("Basic Integration", "Quick rewarded video integration", Intent(this, RewardedVideosActivity::class.java)),
            DemoMenuItem("Zone Integration", "Create different user experiences of the same ad type", Intent(this, RewardedVideosZoneActivity::class.java))
    )
}
